src/js/media-play-button.ts

```diff
@@ -53,13 +87,32 @@ class MediaPlayButton extends MediaChromeButton {
     ];
   }
 
+  tooltip: HTMLElement;
+
   constructor(options = {}) {
     super({ slotTemplate, ...options });
+    this.tooltip = this.shadowRoot.querySelector('#tooltip');
   }
 
   connectedCallback(): void {
     updateAriaLabel(this);
     super.connectedCallback();
+    this.addEventListener(
+      'mouseenter',
+      updateTooltipPosition.bind(null, this.tooltip, 'media-control-bar')
```

Comment:
Buttons won't always be contained by a control bar, if that's required here.

Reply:
Won't be a hard requirement, this is just for testing out the logic of it being contained by "something", as we know that's needed for Mux Player. How and when this is invoked isn't defined yet